hosts_example.yml

```diff
@@ -189,10 +189,25 @@ all:
     # rbac_component_additional_system_admins:
     #   - user1
     #   - user2
+    ##
+    ####  Configuring Role Based Access Control with a remote MDS ####
+    ## To have CP-Ansible configure Brokers and Components for RBAC with the MDS on a remote cluster, set these mandatory variables in addition to those listed above:
+    # rbac_enabled: true
+    # external_mds_enabled: true
+    # external_super_user: mds
+    # mds_ssl_enabled: true
+    # mds_broker_bootstrap_servers: <mds-broker-hostname><port>,<mds-broker-hostname><port>
+    # mds_bootstrap_server_urls: http(s)://<mds-broker-hostname>:8090,http(s)://<mds-broker-hostname>:8090
+    # mds_broker_listener:
+    #   ssl_enabled: true <set to false if remote MDS does not use TLS>
+    #   ssl_mutual_auth_enabled: true <set to false if remote MDS doe not use MTLS>
+    #   sasl_protocol: none <set protocol for remote MDS, options are: kerberos, sasl_plain, sasl_scram>
+    ##
```

Comment:
maybe some docs on these here ie whats the difference between the bootstrap_servers and server_urls in laymans terms